src/components/Header/Header.tsx

```diff
@@ -10,8 +18,72 @@ function Header() {
   const setActive = ({ isActive }: { isActive: boolean }) =>
     isActive ? active : '';
 
+  const { header, info, info__left, info__right, line, main } = classes;
+  const { main__top, main__bottom, main__logo, main__search } = classes;
+  const { main__categories, main__divider, main__input } = classes;
+  const { main__icons, main__cart, info__link, info__info } = classes;

```

Comment:
How many lines will you have ? 
Don't use destructuring, when you have a lot of classes